### ISSUE TITLE
iNaturalist pull operations: how to distinguish Nests from Specimens

### ISSUE THREAD

**Issue by niconoe** [inbo/vespa-watch#31]
(This is for existing Vespa observations @iNaturalist, since the vespa-watch observations are never pulled)

I guess 99% of those will be Specimen observation so for the moment I sets the type to "Specimen" for all of them. Should we continue like that or improve this process?

One possible improvement would be:
- We (manually) take the habit to assign a specific observation field @iNaturalist to those observations, something like is_nest=True
- We pulling observations to our database, we assume it's a specimen except if is_nest=True

@peterdesmet , others, any opinion?

**Comment by peterdesmet** [comment]
> Vespa-watch observations are never pulled

They are for the scientific name, see #2 

- I would call them "individual", rather than "specimen"
- You can assume "individual" for all observations pulled
- In a later stage, "nest" vs "individual" will be assigned depending on an observation field in inaturalist, with "individual" still as default

**Comment by timadriaens** [comment]
indeed, would be good to add "nest" to the survey annotation requirement in iNaturalist, as would the plant species on which individuals are observed (associated taxon?) but this can be done at a later stage (I don't directly see how I should do this in the edit mode on the survey actually).

**Comment by niconoe** [comment]
Thanks all!

A new observation field (is_nest) has been created at iNat so we can flag some observation in it:https://www.inaturalist.org/observation_fields/971.

We take care of this observation field when pulling observations.

Example: https://www.inaturalist.org/taxa/119019-Vespa-velutina

**Comment by niconoe** [comment]
I noticed some observations by Karel already have a "nest" tag. I prefer that we use the observation field for this, since tags have restrictions (cannot be put on observation from other people, for example)

**Comment by niconoe** [comment]
Note: we also have observations that are mainly for a nest, but where an individual captured is also shown (several pics). What to do with those?

**Comment by niconoe** [comment]
Okay, we now use the is_nest observation field for this

**Comment by KarelSchoonvaere** [comment]
Hi,
That's right, I was using tags to specify the category of observations. There are 3 categories in my custom database: Nest, Hive and Individual. The Hive tag is given to observations from beekeepers that report an attack on their honey bee hive. The Individual tag is a generic tag for all loose observations in the field. Could we extend this categorization in iNaturalist? For us it would be a more value to have all beekeepers with attacks reported. 

How are the destroyed nests distinguished from the undestroyed nests? 

In attachement my custom database (in .xlsx) with all observations uptil now. I have tried to upload them in batch by converting to .csv but it keeps giving an error. Can someone help me how to sort this out? These do not include recurring observations from the same observer over time, as in waarnemingen.be

[inaturalist_uploads20181020.xlsx](https://github.com/inbo/vespa-watch/files/2502287/inaturalist_uploads20181020.xlsx)

Thanks!

**Comment by peterdesmet** [comment]
@KarelSchoonvaere: let’s create a separate issue for discussing uploading your observations to iNat.

@niconoe: rather than a binary field, is_nest, I would go for a controlled text field. Not sure about the name though. “evidence” or “vespawatch_evidence” or other? A Darwin Core field doesn’t come to mind, except occurrenceRemarks.

**Comment by niconoe** [comment]
@peterdesmet : allright, I suggest vespawatch_evidence so our stuff is namespaced! The accepted values would be nest/hive attack/individual?

In that case, should we also add "hive attacks" as a category on our side (since we already have nest/individuals)?

**Comment by peterdesmet** [comment]
Ok! And ok for accepted values: will see if that works out as we want.

**Comment by peterdesmet** [comment]
@KarelSchoonvaere, revisiting this issue. We would keep the type of observations to `nest` and `individual`. "attack on hive"  can be reported for both individuals and nests in the `behaviour` field, which is more fitting for this information.

**Comment by niconoe** [comment]
TODO:

- [ ] Remove the `is_nest` observation field @iNaturalist
- [ ] Create the `vespawatch_evidence` field to replace if (accepted values: nest | individual)
- [ ] Update the push/pull code so it uses the new field appropriately.

**Comment by niconoe** [comment]
vespawatch_evidence field ID is: 9770
